### PR TITLE
more accurate description of the config parameters

### PR DESCRIPTION
**Pull request by dindinw**

### CODE REVIEW

File: example.pool.conf [hunk 1]
```diff
@@ -38,7 +38,12 @@ log_level=info
 
 #### Cuckoo Config ####
 # edge bits (24)
+# WARNING, the 'edge_bits' works for cuckaroo ONLY at the current stage.
```
Comment (jamesvan2019) [inline example.pool.conf]: the edg_bits use in too https://github.com/Qitmeer/qitmeer-miner/pull/112,so it useful for cuckoo

File: example.solo.conf [hunk 2]
```diff
@@ -46,7 +46,12 @@ log_level=debug
 
 #### Cuckoo Config ####
 # edge bits (24)
+# WARNING, the 'edge_bits' works for cuckaroo ONLY at the current stage.
```
Comment (jamesvan2019) [inline example.solo.conf]: the edg_bits use in too https://github.com/Qitmeer/qitmeer-miner/pull/112,so it useful for cuckoo

File: example.pool.conf (hunk 1 above)
Comment (dindinw) [reply example.pool.conf]: fixed

File: example.solo.conf (hunk 2 above)
Comment (dindinw) [reply example.solo.conf]: fixed

**Pull request merged**